### PR TITLE
Add content/es/docs/reference/glossary/selector.md

### PR DESCRIPTION
**Pull request by slienyp**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 
> For Chinese localization, base branch to release-1.14
>
> For Korean Localization: set Base Branch to dev-1.14-ko.\<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/es/docs/reference/glossary/selector.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+aka: 
+tags:
+- fundamental
+---
+ Permite a los usuarios filtrar recursos por labels.
+
+<!--more--> 
+
+Los selectors se aplican al realizar consultas de listas de recursos para filtrar por {{< glossary_tooltip text="Labels" term_id="label" >}}.
```
Comment (raelga) [inline content/es/docs/reference/glossary/selector.md]:
```suggestion
Los Selectors se aplican al realizar consultas de listas de recursos para filtrar por {{< glossary_tooltip text="Labels" term_id="label" >}}.
```

Con S mayúscula ya que se trata de un concepto/objeto de Kubernetes.

File: content/es/docs/reference/glossary/selector.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+date: 2018-04-12
+full_link: /docs/concepts/overview/working-with-objects/labels/
+short_description: >
+  Permite a los usuarios filtrar recursos por labels.
+
+aka: 
+tags:
+- fundamental
+---
+ Permite a los usuarios filtrar recursos por labels.
```
Comment (raelga) [inline content/es/docs/reference/glossary/selector.md]:
```suggestion
 Permite a los usuarios filtrar recursos por Labels.
```

Con L mayúscula ya que se trata de un concepto/objeto de Kubernetes.

File: content/es/docs/reference/glossary/selector.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+title: Selector
+id: selector
+date: 2018-04-12
+full_link: /docs/concepts/overview/working-with-objects/labels/
+short_description: >
+  Permite a los usuarios filtrar recursos por labels.
```
Comment (raelga) [inline content/es/docs/reference/glossary/selector.md]:
```suggestion
  Permite a los usuarios filtrar recursos por Labels.
```

Con L mayúscula ya que se trata de un concepto/objeto de Kubernetes.

File: content/es/docs/reference/glossary/selector.md (hunk 1 above)
Comment (electrocucaracha) [inline content/es/docs/reference/glossary/selector.md]:
```suggestion
Los Selectors se aplican al realizar consultas de listas de recursos para ser filtrados por {{< glossary_tooltip text="Labels" term_id="label" >}}.
```
Que opinan?

**Comment by raelga** [comment]
Buenas @slienyp has podido revisar los cambios?

Muchas gracias!

**Comment by raelga** [comment]
Muchas gracias @slienyp !!

Sólo faltaría el squash y que @electrocucaracha confirme la resolución de su review!

🎉 💙

**Comment by raelga** [comment]
Muchas gracias!

/lgtm
/approve

**Pull request merged**